pkg/image/registry/imagerepositorytag/rest.go

```diff
@@ -0,0 +1,88 @@
+package imagerepositorytag
+
+import (
+	"fmt"
+	"strings"
+
+	kapi "github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/api/errors"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/apiserver"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/labels"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/runtime"
+
+	"github.com/openshift/origin/pkg/image/api"
+	"github.com/openshift/origin/pkg/image/registry/image"
+	"github.com/openshift/origin/pkg/image/registry/imagerepository"
+)
+
+// REST implements the RESTStorage interface in terms of an Registry and Registry.
+// It Only supports the Create method and is used to simply adding a new Image and tag to an ImageRepository.
```

Comment:
Actually, this should be updated to say that it only supports Get and that it's for retrieving an Image by name & tag.